media/js/app/panel.js

```diff
@@ -57,23 +68,6 @@
             jQuery(self.el).css('height', visible + 'px');
         };
 
```

Comment:
Thanks for looking into this. I'm not sure what led me to change that call to `self.loadTemplates(0)` to using `MediaThread.loadTemplates()` - maybe the goal was consolidating redundant code. I did notice the `newPanel` stuff.. anyways, I'm investigating now.